internal/pkg/cli/flag.go

```diff
@@ -124,14 +124,14 @@ Defaults to all logs. Only one of end-time / follow may be used.`
 	storageServiceFlagDescription = "Name of the service to associate with storage."
 
 	numFlagDescription              = "Optional. The number of tasks to set up. Default to 1."
-	cpuFlagDescription              = "Optional. The number of cpu-units to reserve. Default to 256."
+	cpuFlagDescription              = "Optional. The number of CPU units to reserve. Default is 256 (1/4 vCPU)."
 	memoryFlagDescription           = "Optional. The amount of memory to reserve. Default to 512."
 	imageFlagDescription            = "Optional. The image to run instead of building a dockerfile."
 	taskRoleFlagDescription         = "Optional. The role for the task to use."
 	subnetIDFlagDescription         = "Optional. The subnet id for the task to use."
 	securityGroupIDsFlagDescription = "Optional. The security group id(s) for the task to use."
-	envVarsFlagDescription          = "Optional. A key=value list of environment variables"
-	commandFlagDescription          = "Optional. List of commands that are passed to docker run"
+	envVarsFlagDescription          = "Optional. A key=value list of environment variables."
```

Comment:
Does this mean that the flag can be specified more than once? We want our descriptions to be clear enough that we don't need to add usage examples elsewhere.